torch_glow/src/CMakeLists.txt

```diff
@@ -38,6 +38,19 @@ target_link_libraries(PyTorchFileLoader
                       PRIVATE
                         PyTorchModelLoader)
 
+add_library(TorchGlowTraining
```

Comment:
In a lot of the rest of Glow there is one target per directory. I'm not sure this is worth creating a new directory but I do think we're starting to get too many targets here